docs/entities.md

```diff
@@ -573,7 +573,7 @@ You can change it by specifying your own name.
 * `asExpression: string` - Generated column expression. Used only in [MySQL](https://dev.mysql.com/doc/refman/5.7/en/create-table-generated-columns.html).
 * `generatedType: "VIRTUAL"|"STORED"` - Generated column type. Used only in [MySQL](https://dev.mysql.com/doc/refman/5.7/en/create-table-generated-columns.html).
 * `hstoreType: "object"|"string"` - Return type of `HSTORE` column. Returns value as string or as object. Used only in [Postgres](https://www.postgresql.org/docs/9.6/static/hstore.html).
-* `array: boolean` - Used for postgres and cockroachdb column types which can be array (for example int[])
+* `jsonType: "object"|"string"` - Return type of `JSON` or `JSONB` column. Returns value as string or as object. Used only in Postgres* `array: boolean` - Used for postgres and cockroachdb column types which can be array (for example int[])
```

Comment:
missing return line before `array: boolean`

Reply:
Sorry about that.